managed/services/backup/backup_service.go

```diff
@@ -524,6 +524,12 @@ func (s *Service) checkArtifactModePreconditions(ctx context.Context, artifactID
 		return errors.Wrapf(ErrArtifactNotReady, "artifact %q in status: %q", artifactID, artifact.Status)
 	}
 
+	if artifact.IsShardedCluster {
+		return errors.Wrapf(ErrIncompatibleService,
+			"artifact %q was taken from sharded cluster and cannot be restored from UI; "+
```

Comment:
more helpful to link to it directly imo https://docs.percona.com/percona-monitoring-and-management/get-started/backup/backup_mongo.html (or something more helpful).